src/test/scala/nl.knaw.dans.easy.solr4files/components/DDMSpec.scala

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.solr4files.components
+
+import java.io.FileInputStream
+import java.net.{ HttpURLConnection, URL }
+
+import org.scalatest.{ FlatSpec, Matchers }
+
+import scala.util.Try
+import scala.xml.XML
+
+class DDMSpec extends FlatSpec with Matchers {
+
+  "solrLiteral" should "return proper values" in {
+    assume(canConnectToEasySchemas)
+    val ddm = new DDM(resource.managed(new FileInputStream(
+      "src/test/resources/vault/stores/pdbs/bags/9da0541a-d2c8-432e-8129-979a9830b427/metadata/dataset.xml"
+    )).acquireAndGet(XML.load))
+    ddm.accessRights shouldBe "OPEN_ACCESS"
+    val literals: Seq[(String, String)] = ddm.solrLiterals
+      .map { case (k, v) => (k, v.replaceAll("\\s+", " ").trim) }
+      .filter { case (_, v) => !v.isEmpty }
+    val expected = Seq(
+      ("dataset_doi", "10.5072/dans-x6g-x2hb"),
+      ("dataset_narcis_audience", "D30000"),
+      ("dataset_audience", "Humanities"),
+      ("dataset_relation", "/domain/dans/user/janvanmansum/collection/Jans-test-files/presentation/easy-dataset:14"),
+      ("dataset_relation", "http://x"),
+      ("dataset_relation", "dummy"),
+      ("dataset_relation", "blabla"),
+      ("dataset_coverage", "random text"),
+      ("dataset_creator", "Captain J.T. Kirk United Federation of Planets"),
+      ("dataset_subject", "astronomie"),
+      ("dataset_subject", "ruimtevaart"),
+      ("dataset_abr_subject", "IX"),
+      ("dataset_coverage", "PALEOLB"),
+      ("dataset_subject", "Infrastructuur, onbepaald"),
+      ("dataset_title", "Reis naar Centaur-planetoïde"),
+      ("dataset_title", "Trip to Centaur asteroid")
+    )
+    // verbose check to figure out which one is wrong in case of problems
+    expected.foreach(literals should contain(_))
```

Comment:
`expected should contain theSameElementsAs literals`?
or maybe `theSameElementsInOrderAs`?

Reply:
How can I clarify the comment?